ezbake/config/conf.d/auth.conf

```diff
@@ -10,7 +10,18 @@ authorization: {
             }
             allow: "$1"
             sort-order: 500
-            name: "puppetlabs catalog"
+            name: "puppetlabs v3 catalog from agents"
+        },
+        {
+            # Allow services to retrieve catalogs on behalf of others
+            match-request: {
+                path: "^/puppet/v4/catalog/?$"
+                type: regex
+                method: [get, post]
```

Comment:
So I think right now if you do a GET it will always return 404. Do we want to include GET here?

Reply:
Good catch, no we probably shouldn't do that.